wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/id.md

```diff
@@ -52,14 +47,14 @@ Seseorang baru saja memposting beberapa saran di beatmap milikmu, kerja bagus! -
 Terkadang, satu mod dan satu tanggapan saja tidak cukup untuk membuat semua pihak menyetujui jika map sudah selesai masalahnya. Sewaktu-waktu mapper bisa saja tidak setuju dengan saran yang diberikan, dan modder juga bisa saja tidak puas dengan tanggapan yang diberikan. Pertimbangan lain, kedua belah pihak bisa saja setuju bahwa ada satu opsi lebih baik, namun bukan berarti opsi yang dipilih meruakan yang terbaik dan masih bisa ditingkatkan lagi dengan masukan tambahan. Umpan balik berkelanjutan adalah cara terbaik dalam menyempurnakan beatmap, jadi libatkan orang-orang dan coba bicarakan secara seksama untuk mendapatkan ide lain, atau lihat semua hal dari berbagai sudut pandang yang diperlukan.
 
 - Jika kamu belum yakin mengenai suatu hal, minta orang tersebut untuk menjelaskan apa maksudnya. Mereka harus lebih dari siap untuk menjelaskan berbagai hal kepadamu lebih rinci.
-- Masih belum bisa mencapai kesepakatan? Minta opini kepada seseorang yang memiliki lebih banyak pengalaman, seperti anggota BN atau NAT. Tetapi ingatlah, jadwal mereka padat! Sadarlah akan banyaknya alternatif yang lebih cocok dibanding sesuatu yang kamu sarankan sebelumnya, jangan takut untuk mengesampingkan ego dirimu - **terkadang melakukan kesalahan bisa jauh lebih berharga ilmu dan pengalamannya dibanding menjadi benar setiap saat**.
+- Masih belum bisa mencapai kesepakatan? Minta opini kepada seseorang yang memiliki lebih banyak pengalaman, seperti anggota [BN](/wiki/People/The_Team/Beatmap_Nominators) atau [NAT](/wiki/People/The_Team/Nomination_Assessment_Team). Tetapi ingatlah, jadwal mereka padat! Sadarlah akan banyaknya alternatif yang lebih cocok dibanding sesuatu yang kamu sarankan sebelumnya, jangan takut untuk mengesampingkan ego dirimu - **terkadang melakukan kesalahan bisa jauh lebih berharga ilmu dan pengalamannya dibanding menjadi benar setiap saat**.
```

Comment:
```suggestion
- Masih belum bisa mencapai kesepakatan? Minta opini kepada seseorang yang memiliki lebih banyak pengalaman, seperti anggota [BN](/wiki/People/The_Team/Beatmap_Nominators) atau [NAT](/wiki/People/The_Team/Nomination_Assessment_Team). Tetapi ingatlah, jadwal mereka padat! Sadarlah akan banyaknya alternatif yang lebih cocok dibanding sesuatu yang kamu sarankan sebelumnya, jangan takut untuk mengesampingkan ego dirimu - **terkadang kesalahan bisa jauh lebih berharga dibanding kebenaran.**.
```

Reply:
imo it's better to reword this to be like

`terkadang kesalahan dapat menjadi guru yang paling berharga.`